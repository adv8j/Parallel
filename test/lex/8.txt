@Mem
{
    R2->R3;
}
