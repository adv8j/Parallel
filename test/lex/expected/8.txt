MEM
LBRACE
IDENTIFIER
ARROW
IDENTIFIER
SEMICOLON
RBRACE
